Packs/ServiceNowv2/Integrations/ServiceNowv2/ServiceNowv2.py

```diff
@@ -0,0 +1,1542 @@
+    },
+    'sc_request': {
+        '1': '1 - Approved',
+        '3': '3 - Closed',
+        '4': '4 - Rejected'
+    }
+}
+
+TICKET_PRIORITY = {
+    '1': '1 - Critical',
+    '2': '2 - High',
+    '3': '3 - Moderate',
+    '4': '4 - Low',
+    '5': '5 - Planning'
+}
+
+SNOW_ARGS = ['active', 'activity_due', 'opened_at', 'short_description', 'additional_assignee_list', 'approval_history',
+             'approval_set', 'assigned_to', 'assignment_group',
+             'business_duration', 'business_service', 'business_stc', 'calendar_duration', 'calendar_stc', 'caller_id',
+             'caused_by', 'close_code', 'close_notes',
+             'closed_at', 'closed_by', 'cmdb_ci', 'comments', 'comments_and_work_notes', 'company', 'contact_type',
+             'correlation_display', 'correlation_id',
+             'delivery_plan', 'delivery_task', 'description', 'due_date', 'expected_start', 'follow_up', 'group_list',
+             'hold_reason', 'impact', 'incident_state',
+             'knowledge', 'location', 'made_sla', 'notify', 'order', 'parent', 'parent_incident', 'priority',
+             'problem_id', 'resolved_at', 'resolved_by', 'rfc',
+             'severity', 'sla_due', 'state', 'subcategory', 'sys_tags', 'time_worked', 'urgency', 'user_input',
+             'watch_list', 'work_end', 'work_notes', 'work_notes_list',
+             'work_start', 'impact', 'incident_state', 'title', 'type', 'change_type', 'category', 'state', 'caller']
+
+# Every table in ServiceNow should have those fields
+DEFAULT_RECORD_FIELDS = {
+    'sys_id': 'ID',
+    'sys_updated_by': 'UpdatedBy',
+    'sys_updated_on': 'UpdatedAt',
+    'sys_created_by': 'CreatedBy',
+    'sys_created_on': 'CreatedAt'
+}
+
+
+def get_server_url(server_url: str) -> str:
+    url = server_url
+    url = re.sub('/[/]+$/', '', url)
+    url = re.sub('/$', '', url)
+    return url
+
+
+def create_ticket_context(data: dict) -> Any:
+    """Create ticket context.
+
+    Args:
+        data: ticket data.
+
+    Returns:
+        ticket context.
+    """
+    context = {
+        'ID': data.get('sys_id'),
+        'Summary': data.get('short_description'),
+        'Number': data.get('number'),
+        'CreatedOn': data.get('sys_created_on'),
+        'Active': data.get('active'),
+        'AdditionalComments': data.get('comments'),
+        'CloseCode': data.get('close_code'),
+        'OpenedAt': data.get('opened_at')
+    }
+
+    # These fields refer to records in the database, the value is their system ID.
+    if 'closed_by' in data:
+        context['ResolvedBy'] = data['closed_by']['value'] if 'value' in data['closed_by'] else ''
+    if 'opened_by' in data:
+        context['OpenedBy'] = data['opened_by']['value'] if 'value' in data['opened_by'] else ''
+        context['Creator'] = data['opened_by']['value'] if 'value' in data['opened_by'] else ''
+    if 'assigned_to' in data:
+        context['Assignee'] = data['assigned_to']['value'] if 'value' in data['assigned_to'] else ''
```

Comment:
we can use `data['closed_by'].get('value, '')` and drop al the last if statements.